### PR TITLE
Aravis capture usb mode option.

### PR DESCRIPTION
**Pull request by hgaiser**

### CODE REVIEW

File: aravis/Cargo.toml [hunk 1]
```diff
@@ -15,12 +15,22 @@ documentation = "https://docs.rs/aravis"
...
+v0_8_3 = []
+v0_8_6 = ["v0_8_3"]
+v0_8_8 = ["v0_8_6"]
+v0_8_11 = ["v0_8_8"]
+v0_8_15 = ["v0_8_11"]
+v0_8_16 = ["v0_8_15"]
+v0_8_17 = ["v0_8_16"]
+v0_8_19 = ["v0_8_17"]
+v0_8_20 = ["v0_8_19"]
+v0_8_22 = ["v0_8_20"]
```
Comment (de-vri-es) [inline aravis/Cargo.toml:27]: Could you add a changelog entry for this too?

**Review by de-vri-es — COMMENTED** [review]
Looks good, just one request regarding the changelog.

File: aravis/Cargo.toml (hunk 1 above)
Comment (hgaiser) [reply aravis/Cargo.toml:27]: Done :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+main:
+  * Add `--usb-mode` option to `aravis-capture`.
+  * Expose feature sets from `aravis-sys` in `aravis`.
```
Comment (de-vri-es) [inline CHANGELOG.md]:
```suggestion
  * Expose features from `aravis-sys` in `aravis`.
```

**Review by de-vri-es — APPROVED** [review]

**Pull request merged**